src/content/wiki/tutorials/schematics.mdoc

```diff
@@ -46,23 +47,39 @@ Schematics are files containing block and entity information of a certain area a
     - [How to create parent/child buildings or decorations?](#how-to-create-parentchild-buildings-or-decorations)
   - [What if I have another question?](#what-if-i-have-another-question)
 
+## A Schematic World
+
+While you can build and scan schematics in any world, it's strongly recommended to create a separate creative world to design and build them, separate from your regular colony worlds. It's also often easier to design and build in single-player, though you can of course do it on a server if you want to collaborate with others. It's also recommended to do your builds outside of any existing colony (typically you would not create any colonies at all in that world) and even to disable mobs/weather/ticks to reduce disturbance of your builds.
+
+If you're designing a full style pack, then on the [Discord server](https://discord.minecolonies.com/) you can find a player-created world download that contains build "pads" for most of the building types, along with a command-block-based auto-scanning system to help speed things up -- though of course you can do things a different way if you prefer.
+
+{% version name="1.19" after=true %}
+It's recommended to enable the `blueprintbuildmode` setting in the MineColonies server config file; this makes it a bit easier to do certain things without creating a colony.
+{% /version %}
+
+If you're a Patreon, you have access to the official schematic server, and can request a plot area to make your own blueprints there, which may be useful if you want to collaborate with other Patreons.
+
 ## Scanning a New Structure
 
 Once you have a structure or area you want to scan to have your Builder build, you need to determine the exact area that needs to be scanned. 
 
+If the area you want to scan contains multiple anchor blocks (hut blocks, tag anchors, or decoration controllers; also including some special blocks like the stash) you should shift-left-click the correct anchor block (e.g. the barracks hut block in a barracks schematic) first, before selecting corners.
+
 ![TestBuilding](@assets/images/wiki/tutorial/testbuilding.png)
 
-Take your {% item name="placeholderblocks" /%} in the top right corner and right-click on it.
+Using the {% item name="scan_tool" /%}, left-click on the bottom-left corner of your blueprint.
 
 ![ScanPoint1](@assets/images/wiki/tutorial/scan1.png)
 
+Next, right-click on the top-right corner of your blueprint. It's often helpful to place a {% item name="placeholderblocks" %}light placeholder{% /item %} there so that you have something convenient to click on, without disturbing your actual build.
```

Comment:
The discord (and the previous version of the wiki tutorial) pretty much universally calls it a "light placeholder" for clarification purposes, since "placeholder" is potentially confusable with the other types of placeholder.  Technically I guess the placeholder page itself calls it a "tan" placeholder instead.

> I'd rather provide a way to show the name of the provided item but still linking to the combined page. (aka placeholderblocks)

If you do, that would be fine, and I'd change this to use that.  I'm not sure how to make it do that myself.